### PR TITLE
Adding aarch64 instance types for azure

### PR DESCRIPTION
**Pull request by kelbrown20**
For version: 4.12+

Description: Created a new file to use in the "Tested instance types section" for Azure.  An example of the formating can be found in [modules/installation-aws-tested-machine-types.adoc](https://github.com/openshift/openshift-docs/blob/main/modules/installation-aws-tested-machine-types.adoc)
[OSDOCS-3329](https://issues.redhat.com/browse/OSDOCS-3329)  

Note: There are likely more instance types that need to be added here

@Prashanth684 Can you PTAL at this PR when you get the chance? Thank you!

### CODE REVIEW

File: docs/user/azure/tested_instance_types_aarch64.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+* `standardD4psv5`
+* `standardD8psv5`
```
Comment (Prashanth684) [inline docs/user/azure/tested_instance_types_aarch64.md:2]: we can probably change this to `standardDpsv5` family similar to x86. we could add more arm types. let me see if i can come up with all the arm instances

**Comment by MayXuQQ** [comment]
@lwan-wanglin whether all the instance type have been verified on arm ? thanks

**Comment by lwan-wanglin** [comment]
> @lwan-wanglin whether all the instance type have been verified on arm ? thanks

Thanks @MayXuQQ , we only verified one instance type on arm now, will launch jobs to verify all instance types listed and give a feekback.

**Comment by lwan-wanglin** [comment]
Verified all listed arm instance types
/lgtm

**Comment by MayXuQQ** [comment]
@lwan-wanglin the family name is not same with az command return. 
suggest: 

standardDPSv5Family
standardDPDSv5Family
standardDPLDSv5Family
standardDPLSv5Family
standardEPSv5Family
standardEPDSv5Family

**Comment by lwan-wanglin** [comment]
> @lwan-wanglin the family name is not same with az command return. suggest:
> 
> standardDPSv5Family standardDPDSv5Family standardDPLDSv5Family standardDPLSv5Family standardEPSv5Family standardEPDSv5Family

Thanks @MayXuQQ,  didn't realize the letter case problems.
@kelbrown20 can we fix it as @MayXuQQ's suggestion? thanks!

**Comment by kelbrown20** [comment]
@MayXuQQ @lwan-wanglin How does this update look?

**Comment by lwan-wanglin** [comment]
> @MayXuQQ @lwan-wanglin How does this update look?

@kelbrown20 sorry for the trouble and thanks for your quick , lgtm!

**Comment by MayXuQQ** [comment]
> 

why add * ? 
ref : https://github.com/openshift/installer/blob/master/docs/user/azure/tested_instance_types.md

**Comment by kelbrown20** [comment]
@patrickdillon This PR should be all set and ready to be merged. Please let me know if there are any further updates that are needed. Thank you!

**Pull request merged**